theories/.dir-locals.el

```diff
@@ -0,0 +1,6 @@
+((coq-mode . ((eval . (let ((default-directory (locate-dominating-file
+						buffer-file-name ".dir-locals.el")))
+			(make-local-variable 'coq-prog-args)
+			(make-local-variable 'coq-prog-name)
+			(setq coq-prog-args `("-emacs" "-coqlib" ,(expand-file-name "..") "-R" ,(expand-file-name ".") "Coq"))
```

Comment:
The coq-prog-name assumes a particular directory structure, which can be overridden when running `configure`.